File: android/app/src/main/kotlin/com/develop/base_flutter/MySecondEventChannel.kt
package com.develop.base_flutter

import io.flutter.plugin.common.EventChannel

class MySecondEventChannel : EventChannel.StreamHandler {

    override fun onListen(arguments: Any?, events: EventChannel.EventSink?) {
        events?.success("${arguments}_1")
        events?.success("${arguments}_2")
        events?.success("${arguments}_3")
    }

    override fun onCancel(arguments: Any?) {

    }
}

File: ios/Runner/MySecondEventChannel.swift
//
//  MySecondEventChannel.swift
//  Runner
//
//  Created by Tung on 24/05/2023.
//

import Foundation

class MySecondEventChannel : NSObject, FlutterStreamHandler {
    public func onListen(withArguments arguments: Any?, eventSink events: @escaping FlutterEventSink) -> FlutterError? {
        events("@_1".replacingOccurrences(of: "@", with: arguments as? String ?? "nil"))
        events("@_2".replacingOccurrences(of: "@", with: arguments as? String ?? "nil"))
        events("@_3".replacingOccurrences(of: "@", with: arguments as? String ?? "nil"))
            return nil
        }

    public func onCancel(withArguments arguments: Any?) -> FlutterError? {
        return nil
    }
}
